draft-ietf-rats-architecture.md

```diff
@@ -143,28 +143,27 @@ Endorsement:
 
 Endorser:
 
-: An entity (typically a manufacturer) whose Endorsements help Verifiers appraise the authenticity of Evidence
+: An entity that creates Endorsements that can be used to help to appraise the trustworthiness of Attesters
 
 Evidence:
 
 : A set of information about an Attester that is to be appraised by a Verifier
 
 Relying Party:
 
-: An entity, that depends on the validity of information about an Attester, for purposes of reliably applying application specific actions.  Compare /relying party/ in {{RFC4949}}
+: An entity that depends on the validity of information about another entity, typically for purposes of authorization.  Compare /relying party/ in {{RFC4949}}
```

Comment:
FYI I am ok either way, but I think "another entity" was previously changed to "an Attester" so not sure why reverting is being proposed in this PR but again I can live with it either way.

Reply:
This change has been reverted.